README.md

```diff
@@ -110,6 +110,8 @@ both for the command line and for other platforms:
 - [Haskell client](https://github.com/psibi/tldr-hs):
   `stack install tldr`
   or `apt-get install tldr` on Debian-based distributions
+- Java clients
+  - [tldr-java-client](https://github.com/seenukarthi/tldr-java-client)
```

Comment:
Sorry for being picky, but I've just taken a fresh look at this, and this may be more appropriate considering that there's only a single Java client at this time:
```suggestion
- [Java client](https://github.com/seenukarthi/tldr-java-client)
```